spec/models/category_spec.rb

```diff
@@ -0,0 +1,46 @@
+# frozen_string_literal: true
+# == Schema Information
+#
+# Table name: categories
+#
+#  id          :integer          not null, primary key
+#  name        :string
+#  description :text
+#  created_at  :datetime
+#  updated_at  :datetime
+#  user_id     :integer
+#  slug        :string
+#
+
+describe Category do
+    context "creation" do
+        it 'creates a valid category' do
+            category = create(:category, :with_user)
+            expect(category).to be_valid
+            expect(Category.count).to eq(1)
+        end
+
+        it 'invalid without a user_id' do
+            category = build(:category, user_id: nil)
+            expect(category).to be_invalid
+        end
+        
+        it 'invalid without a name' do
+            category = build(:category, name: nil)
+            expect(category).to have(1).error_on(:name)
+        end
+    end
+    
+    context "relation" do
+        it 'belongs to a user' do
+            assc = described_class.reflect_on_association(:user)
+            expect(assc.macro).to eq :belongs_to
+        end
+    end
+
+
+  
+
+
+  
+end
```

Comment:
Can we remove a lot of these empty lines and change the indents to only be 2 spaces instead of 4?
https://github.com/ifmeorg/ifme/wiki/Backend-Practices

I feel like this should also include a spec for the class method `link` https://github.com/ifmeorg/ifme/blob/26d231edb2219f7348fc9d7fe7437d9a2fed7087/app/models/category.rb#L22

Other than that LGTM